pod2daemon/Makefile

```diff
@@ -58,15 +68,15 @@ bin/flexvol-armv7: ARCH=armv7
 bin/flexvol-ppc64le: ARCH=ppc64le
 bin/flexvol-s390x: ARCH=s390x
 bin/flexvol-%: $(SRC_FILES)
-	$(DOCKER_RUN) -e CGO_ENABLED=0 $(CALICO_BUILD) go build -v -o bin/flexvol-$(ARCH) flexvol/flexvoldriver.go
+	$(DOCKER_RUN) -e CGO_ENABLED=0 $(CALICO_BUILD) go build -v -o bin/flexvol-$(ARCH) flexvol/flexvoldriver.go $(LDFLAGS)
```

Comment:
This isn't going to work - you'll need to put `LDFLAGS` prior to any golang source files in this command.

You can try it out by running `make build` inside the pod2daemon repository, and you'll see that the build fails.